models/intel/face-detection-adas-0001/model.yml

```diff
@@ -41,4 +41,5 @@ files:
     sha256: a1995292449f55c49116c77c83ef3726e08d19b31ab38cc7a9540116ef8862cf
     source: https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.3/models_bin/2/face-detection-adas-0001/FP16-INT8/face-detection-adas-0001.bin
 framework: dldt
+compilable: yes
```

Comment:
How did you determine which models to mark as compilable? Do the rest of them fail to compile?

Reply:
Many can be compiled, but still some fail. Right now I specified only several models for each demo. I plan to extend number of compilable models in the near future.